change istio  to ambient mode

dfry opened the pull request.

jmutai committed:
Upgrade istio helm version

dfry committed:
feat: add kiali to cc and env

dfry committed:
bump for argocd debug

dfry committed:
fix: change service name

dfry committed:
add prom/grafana endpoints

dfry committed:
fix urls

jmutai committed:
rm configOverride

jmutai committed:
disable other old params

jmutai committed:
Add ambient to cc

jmutai committed:
disable custom image tag

jmutai committed:
Fix istio vs for argocd & keycloak

dfry committed:
fix kiali templates

dfry committed:
add prom/grafana

dfry committed:
bump kiali version

dfry committed:
fix version

dfry committed:
make secret name unique to each pm4ml

dfry committed:
cleanup mtls egress

dfry committed:
change version for DR

jmutai committed:
Add ambient files

jmutai committed:
update istio values

dfry committed:
add istio annotations for ambient

dfry committed:
fix pod annotations

dfry committed:
cleanup mcm and add gateway api

dfry committed:
fix typo on domain

dfry committed:
rm unused role/rb

dfry committed:
missing ns ref

dfry committed:
rm sidecar

dfry committed:
add waypoint label

dfry committed:
first draft add policy for ambient mode on all ns

dfry committed:
modify istio for cc

dfry committed:
add opt out and exclude certain ns

dfry committed:
add exclusions for ambient mode ns

dfry committed:
opt out vault

dfry committed:
opt out vault and gitlab

dfry committed:
apply to correct app

dfry committed:
add to ns

dfry committed:
opt out gateway

dfry committed:
put back ambient

dfry committed:
opt out nb op

dfry committed:
opt netbird op out of mesh

dfry committed:
move to separate ns for moja and kafka

dfry committed:
fixes for patches and strimzi

dfry committed:
rm duplicate block

dfry committed:
fix egress. mtls

dfry committed:
fix oidc kubeconfig name

Commit:
add env var to turn off default policy

Commit:
bump nb to latest

kalinkrustev committed:
fix: add proxy security headers configuration for Istio external gateway

rforgac committed:
feat: add openebs

rforgac committed:
feat: add missing common vars of openebs

rforgac committed:
fix: passing of the variables

rforgac committed:
feat: add missing sync wave

rforgac committed:
fix: namespace of openebs

rforgac committed:
fix: values

rforgac committed:
feat: change default version of openebs to match envs

Commit:
Deploy monitoring in sc (#654)

* comment out metric-server

* comment out grafana operator as well

* update deployment report dashboard

* add sc-monitoring app

* add monitoring_namespace

* add env variables for prometheus

* update network traffic analyser dashboard name and move its folder

* uncomment grafana operator

* disable network policy for all environment tempo components

Commit:
bump provider nb

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/mojaloop/iac-modules?pullRequest=661) <br/>All committers have signed the CLA.

dfry committed:
Merge branch 'feature/storage-cluster' into feat/istio-ambient

dfry committed:
add ztunnel log level default warn

dfry committed:
split env claim into 2

dfry committed:
bump package fix

dfry committed:
bump new package version

dfry committed:
update values for removing default policy

muzammil360 committed:
Fix mimir and tempo inter pod communication with istio ambient (#676)

- Fix mimir and tempo inter pod communication with istio ambient
- Disable NetworkPolicy objects for prometheus stack deployments

dfry committed:
missed template

dfry committed:
alter claims for new package

dfry committed:
bump package for nb

dfry committed:
add empty aws kustomize

jmutai reviewed (approved):
lgtm
